presto-hive/src/main/java/com/facebook/presto/hive/metastore/CachingHiveMetastore.java

```diff
@@ -218,6 +219,17 @@ private CachingHiveMetastore(ExtendedHiveMetastore delegate, ExecutorService exe
                         return loadTablePrivileges(key.getUser(), key.getDatabase(), key.getTable());
                     }
                 }, executor));
+
+        rolesCache = newCacheBuilder(expiresAfterWriteMillis, refreshMills, maximumSize)
+                .build(asyncReloading(new CacheLoader<String, Set<String>>()
```

Comment:
The only valid value for `Void` type is `null`. But `null` is not allowed to be a key in Guava cache.